florist/tests/api/test_launch.py

```diff
@@ -0,0 +1,50 @@
+import os
```

Comment:
I think this is a good opportunity to make an integration test folder as this is looking more like an integration test with all the processes being spun. I think a structure like this make sense but let me know what you think:
```python
florist/
  tests/
    integration/
      api/
        test_launch.py
    unit/
      api/
        test_client.py
```

Reply:
Agreed!